### PR TITLE
python-plum-py: added package

### PR DESCRIPTION
**Pull request by D3vil0p3r**
Depedency of python-exif package that is used for LinkScope tool.

### CODE REVIEW

File: packages/python-plum/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+# This file is based on the AUR package by Caleb Maclennan <caleb@alerque.com>
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:4]:
```suggestion
# This PKGBUILD was originally extracted from AUR.
# AUR Maintainer: Caleb Maclennan <caleb@alerque.com>
```

File: packages/python-plum/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+_pyname=plum
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:6]:
```suggestion
```

File: packages/python-plum/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+pkgname=python-${_pyname,,}
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:7]:
```suggestion
pkgname=python-plum
_pkgname=plum-py
```

https://pypi.org/project/plum-py/

File: packages/python-plum/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+pkgver=0.8.5
+pkgrel=1
+pkgdesc='Pack/Unpack Memory'
+arch=(any)
+url="https://$_pyname-py.readthedocs.io/en/latest/"
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:12]:
```suggestion
url="https://plum-py.readthedocs.io/en/latest/"
```

File: packages/python-plum/PKGBUILD [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+license=(MIT)
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:13]:
```suggestion
license=('MIT')
```

File: packages/python-plum/PKGBUILD [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+depends=(python)
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:14]:
```suggestion
depends=('python')
```

File: packages/python-plum/PKGBUILD [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+makedepends=(python-{build,installer}
+             python-setuptools
+             python-wheel)
+_archive="$_pyname-$pkgver"
+source=("https://gitlab.com/dangass/$_pyname/-/archive/$pkgver/$_archive.tar.bz2")
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:19]: use https://pypi.org/project/plum-py/ and https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-lib-setuptools

File: packages/python-plum/PKGBUILD [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+sha256sums=('41016eed6cdbe1429afc21654cd05f935fc13ae8eb465981f6c3b1e92c57319e')
```
Comment (noraj) [inline packages/python-plum/PKGBUILD:20]: sha512

**Review by noraj — CHANGES_REQUESTED** [review]

File: packages/python-plum/PKGBUILD (hunk 7 above)
Comment (D3vil0p3r) [reply packages/python-plum/PKGBUILD]: Changed. I changed also the pkg name as `python-plum-py`. Is it possible to change the directory of the PKGBUILD as `python-plum-py` by GitHub WebUI?

File: packages/python-plum/PKGBUILD (hunk 8 above)
Comment (D3vil0p3r) [reply packages/python-plum/PKGBUILD]: Done

**Review by D3vil0p3r — COMMENTED** [review]
Applied SHA512 sum

File: packages/python-plum-py/PKGBUILD [hunk 9]
```diff
@@ -0,0 +1,29 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-plum-py
+_pkgname=${pkgname#python-}
+pkgver=0.8.5
+pkgrel=1
+pkgdesc='Pack/Unpack Memory.'
+arch=('any')
+url='https://$_pkgname-py.readthedocs.io/en/latest/'
```
Comment (noraj) [inline packages/python-plum-py/PKGBUILD:10]:
```suggestion
url='https://plum-py.readthedocs.io/en/latest/'
```

**Review by noraj — CHANGES_REQUESTED** [review]

**Pull request merged**